test.sh

```diff
@@ -16,40 +16,52 @@
 
 set -eu -o pipefail
 
+TESTSUITE="${TESTSUITE:-all}" # if env variable not set run all the tests
 exitcode=0
 
-echo "= go mod ================================================================"
-go mod download 2>&1 | grep -v "go: finding" || true
-go mod tidy -v && echo ok || ((exitcode += 2))
-
-echo "= make lint ============================================================="
-make -s lint-ci && echo ok || ((exitcode += 4))
-
-echo "= boilerplate ==========================================================="
-readonly PYTHON=$(type -P python || echo docker run --rm -it -v $(pwd):/minikube -w /minikube python python)
-readonly BDIR="./hack/boilerplate"
-missing="$($PYTHON ${BDIR}/boilerplate.py --rootdir . --boilerplate-dir ${BDIR} | egrep -v '/assets.go|/translations.go|/site/themes/|/site/node_modules|\./out|/hugo/' || true)"
-if [[ -n "${missing}" ]]; then
-    echo "boilerplate missing: $missing"
-    echo "consider running: ${BDIR}/fix.sh"
-    ((exitcode += 4))
-else
-    echo "ok"
+if [ "$TESTSUITE" = "lint" ] || [ "$TESTSUITE" = "all" ]
```

Comment:
nit: Use [[ rather than [: https://google.github.io/styleguide/shell.xml?showone=Test,_%5B_and_%5B%5B#Test,_%5B_and_%5B%5B - here and elsewhere in this file.